src/notification/smartthings.ts

```diff
@@ -0,0 +1,50 @@
+import {SmartThings} from '@bridgerakol/samsung-smart-api';
+import {logger} from '../logger';
+import {config} from '../config';
+
+const {smartthings} = config.notifications;
+
+export async function activateSmartthingsSwitch() {
+  if (
+    ('token' in smartthings &&
+      !smartthings.token &&
+      'device' in smartthings &&
+      !smartthings.device) ||
+    (!('token' in smartthings) && !('device' in smartthings))
+  ) {
+    return;
+  }
+  const st = new SmartThings(smartthings.token);
+  let match = false;
+  try {
+    await st.devices.getList().then(res => {
+      res.data.items.forEach(
+        async (item: {label: string; deviceId: string}) => {
+          if (smartthings.device === item.label) {
+            match = true;
+            let device_status = (await st.devices.getStatus(item.deviceId)).data
+              .components.main.switch.switch.value;
+            if (device_status === 'on') {
+              device_status = true;
+            } else device_status = false;
+            if (!device_status) {
+              logger.debug(`Turning on ${smartthings.device}`);
+              st.devices.commands(item.deviceId, 'on');
```

Comment:
```suggestion
            if (device_status !== 'on') {
              logger.debug(`Turning on ${smartthings.device}`);
              st.devices.commands(item.deviceId, 'on');
```

Can we reduce this to this?